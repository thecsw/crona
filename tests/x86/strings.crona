print : void (s14 : byte array[15]) {
    write s14;
}

main : int() {
    print("lit string arg");
    write "\n";
    input : byte array[48];
    write "What's your name?\n";
    read input;
    write "Hello, ";
    write input;
    write "\n";
    return 0;
}