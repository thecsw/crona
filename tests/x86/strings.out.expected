lit string arg
What's your name?
Hello, Rev. Atkins
